auth/controller_flags_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2025 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package auth_test
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/fluxcd/pkg/auth"
+)
+
+func TestSetDefaultServiceAccount(t *testing.T) {
+	g := NewWithT(t)
+	auth.SetDefaultServiceAccount("test-sa")
+	t.Cleanup(func() {
+		os.Unsetenv(auth.EnvDefaultServiceAccount)
+	})
+	g.Expect(os.Getenv(auth.EnvDefaultServiceAccount)).To(Equal("test-sa"))
+}
+
+func TestSetDefaultKubeconfigServiceAccount(t *testing.T) {
```

Comment:
```suggestion
func TestSetDefaultKubeConfigServiceAccount(t *testing.T) {
```

`Kubeconfig` -> `KubeConfig`

Reply:
😢